libminifi/include/utils/GeneralUtils.h

```diff
@@ -56,6 +56,11 @@ T exchange(T& obj, U&& new_value) {
   obj = std::forward<U>(new_value);
   return old_value;
 }
+
+// partial detection idiom impl, from cppreference.com
```

Comment:
this comment should go back to StringUtils.h

Reply:
Done.